### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by iRON5** [use-global-hook/use-global-hook#73]
What is the status of the project, is this just on maintenance or there are some plans to continue active development?

**Comment by rickstaa** [comment]
@iRON5 I was wondering the same thing when I saw several old pull requests. Based on this issue #66 it looks like the project since May has shared ownership and is maintained. Are there any pull requests you need to have merged or was it like me just a question to see if you should use this hook in your next project?

**Comment by rickstaa** [comment]
@iRON5 Ah, I now see those pull requests are from you. Maybe you can ask @andregardi  or @davityavryan for a review?

**Comment by rickstaa** [comment]
@iRON5 Ah I now see that these pull requests are yours and that you already tagged the organization members.

**Comment by rickstaa** [comment]
@iRON5 Ah, sorry, I now see that these pull requests are yours and that you already tagged the organization members. Maybe let's wait a bit for @andregardi and @davityavryan. If they do not reply, I am happy to help you maintain your fork. I already applied some (i.e. #70 and #72) of your pull request  in my fork.

**Comment by davityavryan** [comment]
Hey @iRON5 @rickstaa 

Answer is no, it's not dead.
My apologies though. I know that we haven't took care of issues and PR-s for some time.
This will change in August (at least for me). I will have more time and will review all issues and PRs.

Planned major changes that I would love to have are:
1. migrate source code to Typescript and providing typings from package itself
2. re-write/improvement of build system for the package
3. test framework introduction (Jest) and some initial coverage (I might need help with this though)
4. documentation improvements and more examples

@andregardi any objections?

**Comment by rickstaa** [comment]
@davityavryan No problem, thanks a lot for taking the time to maintain this project!

**Comment by iRON5** [comment]
@davityavryan  great plans, I'd like to rewrite the lib with TypeScript!
BTW, maybe that makes sense to create github milestones and attach these tickets there

**Comment by yamiteru** [comment]
Looks like the project in fact is dead since nothing has changed as stated in previous comments.

**Comment by iRON5** [comment]
Indeed, my PRs were not reviewed for few months. Maybe that would change someday but the repo has no enough reliability now.

Btw, try the alternatives:
https://github.com/ctrlplusb/easy-peasy
https://github.com/avkonst/hookstate

**Comment by davityavryan** [comment]
My apologies to all. I am going through some rough time currently, personal life-wise. I don't know when I will be able to support this package.

@andregardi can you please add @iRON5 or other interested parties as a co-owner?
You can remove me if needed.

**Comment by andregardi** [comment]
Thank you @davityavryan for looking into that!
I invited @iRON5 to that as an admin.

**Comment by iRON5** [comment]
Ok, thanks guys, I'd like to help with that.
@andregardi but I still need access to npm for publishing updates.
Here is my npm account https://www.npmjs.com/~iron5
